### PR TITLE
Add option to generate sarif output for ingestion into GitHub code scanning results

### PR DESCRIPTION
**Pull request by ochimo**

### CODE REVIEW

**Comment by eacmen** [comment]
@ochimo please address tests that failed in travis

**Comment by ochimo** [comment]
@eacmen linter issues fixed - thanks

File: centrifuge_cli/main.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def echo(self, message, err=False):
 @click.option('--apikey', envvar='CENTRIFUGE_APIKEY', required=True,
               metavar='KEY', help='Your Centrifuge API Key')
 @click.option('--limit', default=20, help='Number of results to return, use -1 for no limit')
-@click.option('--outfmt', default='human', help='Output format of command', type=click.Choice(['human', 'json', 'csv']))
+@click.option('--outfmt', default='human', help='Output format of command', type=click.Choice(['human', 'json', 'csv', 'sarif']))
```
Comment (eacmen) [inline centrifuge_cli/main.py:191]: sarif output format doesn't make sense for all command types. The other output formats are generic and you can output any result from centrifuge to any format. I would add a `--sarif` option to check-policy versus adding sarif as an option here.

**Review by eacmen — CHANGES_REQUESTED** [review]
Looks good overall. Just one issue to address with the outfmt sarif not making sense for all cmdlets. Make the requested changes and I'll approve.

**Review by eacmen — APPROVED** [review]
approved

**Pull request merged**